.github/actions/spelling/excludes.txt

```diff
@@ -0,0 +1,32 @@
+# See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-excludes
+(?:^|/)(?i)COPYRIGHT
+(?:^|/)(?i)LICEN[CS]E
+(?:^|/)package(?:-lock|)\.json$
+(?:^|/)vendor/
+/fonts/
+ignore$
+\.avi$
+\.eot$
+\.ico$
+\.jpe?g$
+\.lock$
+\.map$
+\.min\.
+\.mod$
+\.mp[34]$
+\.png$
+\.svg$
+\.ttf$
+\.wav$
+\.woff$
+\.woff2$
+^docs/_posts/
```

Comment:
```suggestion
```

Let's also check the posts

Reply:
That was there because I chose not to look at them, thinking they might be sacrosanct.

Fwiw, GitHub appears to have had some sort of outage, so your commit half-triggered a run:
https://github.com/jsoref/jekyll/actions/runs/846644548
but it didn't resolve (otherwise, there should have been a comment here by the bot commenting on jsoref/jekyll).

You're going to want to resolve the items listed here:
https://github.com/jekyll/jekyll/runs/2594681160?check_suite_focus=true#step:4:24

It looks like they can all be added to `expect.txt`. You can see the output here:
https://github.com/jekyll/jekyll/pull/8676#issuecomment-841849062

I'm not sure if you've already created a PR to apply the changes it suggests. (Someone did create a PR for 
a03af49...)

I created #8677 in case no one else does.